sycl/doc/extensions/proposed/sycl_ext_oneapi_memcpy2D.asciidoc

```diff
@@ -0,0 +1,137 @@
+= sycl_ext_oneapi_memcpy2D
```

Comment:
```suggestion
= sycl_ext_oneapi_memcpy2d
```

Since you changed the name of the method to use lowercase, change the extension name to match.  Also, change the name of this file to use a lowercase "d".